Product proposal template.md

```diff
@@ -1,30 +1,33 @@
 ## High Level Product Description Template:
 
-In addition to the below, proposed KBase apps should also follow the requirements in the [KBase product requirements document](./KBase%20product%20requirements.md)
+*Proposed KBase apps should also follow the requirements in the [KBase product requirements document](./KBase%20product%20requirements.md). Products that are not apps should follow those requirements where appropriate.*
 
-These high level descriptions must include the following:
+For most types of products, product descriptions should include the following:
 
 ### Summary
-A brief summary of what the product will enable or improve for the user.
+A brief summary of what the product will enable or improve for the user. (In other words, WHY should we develop this product?)
+
+### Keyword(s)
+What areas of science or technology does this product support?
```

Comment:
Think of this as analogous to a "tag" - what parts of KBase (ideally the roadmap) does this affect?